source/wp-content/themes/wporg-showcase-2022/src/site-screenshot/index.php

```diff
@@ -46,7 +46,10 @@ function render( $attributes, $content, $block ) {
 	$post = get_post( $post_ID );
 
 	$screenshot = site_screenshot_src( $post );
-	$screenshot = add_query_arg( 'scale', 2, $screenshot );
+
+	if ( isset( $attributes['useHiRes'] ) && true === $attributes['useHiRes'] ) {
+		$screenshot = add_query_arg( 'scale', 2, $screenshot );
+	}
 
 	$img_content = "<img src='{$screenshot}' alt='" . the_title_attribute( array( 'echo' => false ) ) . "' loading='lazy' />";
```

Comment:
[Sounds like this should be escaped](https://a8c.slack.com/archives/G010HCMN0HF/p1669942609107009)

```suggestion
		$screenshot = add_query_arg( 'scale', 2, $screenshot );
	}

	$img_content = "<img src='" . esc_url( $screenshot ) . "' alt='" . the_title_attribute( array( 'echo' => false ) ) . "' loading='lazy' />";
```